docs/data/material/components/material-icons/material-icons.md

```diff
@@ -14,15 +14,16 @@ The following npm package,
 [@mui/icons-material](https://www.npmjs.com/package/@mui/icons-material),
 includes the 2,100+ official [Material Icons](https://fonts.google.com/icons?icon.set=Material+Icons) converted to [`SvgIcon`](/material-ui/api/svg-icon/) components.
 
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
 :::info
 The `@mui/icons-material` package depends on `@mui/material`, which requires Emotion packages.
+If you don't use Material UI in your project yet, install the icons package with:
+`npm install @mui/icons-material @mui/material @emotion/styled @emotion/react`.
 
-If you don't use Material UI in your project yet, install the icons package with `npm install @mui/icons-material @mui/material @emotion/styled @emotion/react`.  
 See the [Installation](/material-ui/getting-started/installation/) page for additional docs about how to make sure everything is set up correctly.
 :::
 
-{{"component": "modules/components/ComponentLinkHeader.js"}}
-
```

Comment:
I have tried to reduce the vertical space of this at the same time fixing the error raised by the lining rule.